<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="2" total="3" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Amazon Automation Suite" duration-ms="83197" started-at="2021-09-19T22:11:48 CEST" finished-at="2021-09-19T22:13:11 CEST">
    <groups>
    </groups>
    <test name="Amazon Automation Test" duration-ms="83197" started-at="2021-09-19T22:11:48 CEST" finished-at="2021-09-19T22:13:11 CEST">
      <class name="com.amazon.test.AmazonAssignmentTest">
        <test-method status="PASS" signature="beforeSuite()[pri:0, instance:com.amazon.test.AmazonAssignmentTest@691a7f8f]" name="beforeSuite" is-config="true" duration-ms="140" started-at="2021-09-19T22:11:48 CEST" finished-at="2021-09-19T22:11:48 CEST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeSuite -->
        <test-method status="PASS" signature="beforeMethod(java.lang.String,java.lang.reflect.Method)[pri:0, instance:com.amazon.test.AmazonAssignmentTest@691a7f8f]" name="beforeMethod" is-config="true" duration-ms="8398" started-at="2021-09-19T22:11:48 CEST" finished-at="2021-09-19T22:11:56 CEST">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[public void com.amazon.test.AmazonAssignmentTest.testCase3()]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="testCase3()[pri:0, instance:com.amazon.test.AmazonAssignmentTest@691a7f8f]" name="testCase3" duration-ms="73980" started-at="2021-09-19T22:11:56 CEST" finished-at="2021-09-19T22:13:10 CEST">
          <exception class="java.lang.IndexOutOfBoundsException">
            <message>
              <![CDATA[Index: 0, Size: 0]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IndexOutOfBoundsException: Index: 0, Size: 0
at java.util.ArrayList.rangeCheck(Unknown Source)
at java.util.ArrayList.get(Unknown Source)
at com.amazon.pageObjects.AppleWatchesResultPage.print5thFromProductList(AppleWatchesResultPage.java:69)
at com.amazon.test.AmazonAssignmentTest.testCase3(AmazonAssignmentTest.java:61)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IndexOutOfBoundsException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCase3 -->
        <test-method status="PASS" signature="afterMethd(org.testng.ITestResult)[pri:0, instance:com.amazon.test.AmazonAssignmentTest@691a7f8f]" name="afterMethd" is-config="true" duration-ms="785" started-at="2021-09-19T22:13:10 CEST" finished-at="2021-09-19T22:13:11 CEST">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=testCase3 status=FAILURE method=AmazonAssignmentTest.testCase3()[pri:0, instance:com.amazon.test.AmazonAssignmentTest@691a7f8f] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethd -->
        <test-method status="PASS" signature="afterSuite()[pri:0, instance:com.amazon.test.AmazonAssignmentTest@691a7f8f]" name="afterSuite" is-config="true" duration-ms="344" started-at="2021-09-19T22:13:11 CEST" finished-at="2021-09-19T22:13:12 CEST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterSuite -->
      </class> <!-- com.amazon.test.AmazonAssignmentTest -->
    </test> <!-- Amazon Automation Test -->
  </suite> <!-- Amazon Automation Suite -->
</testng-results>
